pennylane/templates/subroutines/qsvt.py

```diff
@@ -328,9 +328,9 @@ def circuit():
 
     if encoding_wires is None or block_encoding is None or "wires" in kwargs.keys():
         warnings.warn(
-            "You may be trying to use the old `qsvt` functionality (now `qml.qsvt_legacy`)."
-            "Make sure you pass a polynomial instead of angles."
-            "Set a value for `block_encoding` to silence this warning.",
+            "You may be trying to use the old `qsvt` functionality (now `qml.qsvt_legacy`).\n"
+            "Make sure you pass a polynomial instead of angles.\n"
+            "Set a value for `block_encoding` to silence this warning.\n",
```

Comment:
Do we need the `f` string prefactor? or does it display properly without it? 

```suggestion
            f"You may be trying to use the old `qsvt` functionality (now `qml.qsvt_legacy`).\n"
            f"Make sure you pass a polynomial instead of angles.\n"
            f"Set a value for `block_encoding` to silence this warning.\n",
```